nUpdate.Test/UseDynamicUriTest.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using nUpdate.Updating;
+
+namespace nUpdate.Test
+{
+    [TestClass]
+    public class UseDynamicUriTest
```

Comment:
Please rename this to "UseDynamicUpdateUriTest".

Reply:
Renamed